transport_analysis/tests/test_velocityautocorr.py

```diff
@@ -42,14 +49,29 @@ def vacf(ag):
 def step_vtraj(NSTEP):
     v = np.arange(NSTEP)
     velocities = np.vstack([v, v, v]).T
-    # NSTEP frames x 1 atom x 3 dimensions, where NSTEP = 5000
+    # NSTEP frames x 1 atom x 3 dimensions, where NSTEP = 5001
     velocities_reshape = velocities.reshape([NSTEP, 1, 3])
     u = mda.Universe.empty(1, n_frames=NSTEP, velocities=True)
     for i, ts in enumerate(u.trajectory):
         u.atoms.velocities = velocities_reshape[i]
     return u
 
 
+# Position trajectory corresponding to unit velocity trajectory
+@pytest.fixture(scope="module")
+def step_vtraj_pos(NSTEP):
+    x = np.arange(NSTEP).astype(np.float64)
```

Comment:
Can we make the name here a bit more descriptive? Maybe `unit_velocity_traj`

Reply:
Does this matter enough to be worth the effort? It's used in a lot of places, so the change would be a little messy.